modules/cli/pages/index.adoc

```diff
@@ -506,27 +506,48 @@ $ skupper gateway generate-bundle <config-filename> <destination-directory>
 
 where:
 
-* <config-filename> - the name of the YAML file, include suffix, that you generated in the previous step.
+* <config-filename> - the name of the YAML file, including suffix, that you generated in the previous step.
 * <destination-directory> - the location where you want to save the resulting gateway bundle, for example `~/gateways`.
 
---
+For example:
+----
+$ skupper gateway generate-bundle database.yaml ./
+----
 
+This bundle contains the gateway definition YAML and a  certificate that allow access to the {service-network}.
 
+--
 
 . Copy the gateway definition file, for example, `mylaptop-jdoe.tar.gz` to the machine that hosts the service you want to expose on the {service-network}.
 
 . From the machine that hosts the service you want to expose:
 +
+--
 ----
 $ mkdir gateway
-$ tar -xvf <gateway-definition-file>
+
+$ tar -xvf <gateway-definition-file> --directory gateway
 $ cd gateway
 $ sh ./launch.py
```

Comment:
or alternatively

$ chmod +x launch.py
$ ./launch.py

Reply:
I think the `sh` option is better (and answers my question in email)